webapp/src/components/sidebar/sidebarAddBoardMenu.tsx

```diff
@@ -115,6 +101,51 @@ class SidebarAddBoardMenu extends React.Component<Props> {
         )
     }
 
+    private renderBoardTemplate(boardTemplate: Board, isGlobal = false): JSX.Element {
```

Comment:
We can do it later, but probably this is a good candidate to create another small component.